### PR TITLE
PR bot improvements

### PR DESCRIPTION
**Pull request by Cyberboss**
@MrStonedOne 
I should have done this in the first place

I explicitly left out the close* keywords because people use those to close open prs

### CODE REVIEW

**Comment by Cyberboss** [comment]
There's a bug in the announcer where it's sending push events to the server

File: tools/github_webhook_processor.php [hunk 1]
```diff
@@ -190,11 +195,11 @@ function handle_pr($payload) {
 	switch ($payload["action"]) {
 		case 'opened':
 			tag_pr($payload, true);
-			break;
+			return;
```
Comment (MrStonedOne) [inline tools/github_webhook_processor.php]: this one should still be a break

**Comment by Cyberboss** [comment]
Fixed

**Comment by Cyberboss** [comment]
@MrStonedOne gonna need another update, that last idea was terrible

**Comment by Supermichael777** [comment]
i feel this will be abused by some of our filthier memers

**Comment by MrStonedOne** [comment]
na, i like it, i'll just keep it in

**Comment by Cyberboss** [comment]
faceroll

**Pull request merged**